src/kakarot/account.cairo

```diff
@@ -177,10 +114,15 @@ namespace Account {
         alloc_locals;
         let starknet_account_exists = is_registered(address.evm);
 
+        let balance = read_balance(address);
+        tempvar balance_ptr = new Uint256(balance.low, balance.high);
```

Comment:
why do we need a `new` keyword here?

Reply:
this is the easy way to make it a pointer. Otherwise I need to use the `__fp__` variable to do `&balance`, which I can do if you prefer

Reply:
no preference! was just curious